### PR TITLE
Add yarn lint to GitHub Actions

### PR DESCRIPTION
**Pull request by dchiquito**
We are now running `tox` in CI, but since the client code also lives in CI, we should also be running `yarn lint`.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -47,3 +47,16 @@ jobs:
           DJANGO_MINIO_STORAGE_ENDPOINT: localhost:9000
           DJANGO_MINIO_STORAGE_ACCESS_KEY: minioAccessKey
           DJANGO_MINIO_STORAGE_SECRET_KEY: minioSecretKey
+  test-ci:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: client
+    steps:
+      -
```
Comment (zachmullen) [inline .github/workflows/ci.yml:56]: ^ This guy right here

Comment (dchiquito) [reply .github/workflows/ci.yml:56]:
```suggestion
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -47,3 +47,15 @@ jobs:
           DJANGO_MINIO_STORAGE_ENDPOINT: localhost:9000
           DJANGO_MINIO_STORAGE_ACCESS_KEY: minioAccessKey
           DJANGO_MINIO_STORAGE_SECRET_KEY: minioSecretKey
+  test-ci:
```
Comment (dchiquito) [inline .github/workflows/ci.yml:50]:
```suggestion
  test-client:
```

**Comment by dchiquito** [comment]
The error is fixed in https://github.com/OpenImaging/miqa/pull/79/commits/58c844ac2fe0514f5f38b24ef75180a59ce1eb33, I'm gonna merge this

**Pull request merged**